src/common

```diff
@@ -101,17 +101,19 @@ else ifeq ($(BUILD_ENV_),Cygwin)
   CXXFLAGS ?= -Wall -O2
   CP_CFLAGS = -MMD -MP
   CP_CXXFLAGS += -MMD -MP -std=c++11 -U__STRICT_ANSI__
+  # Cygwin-g++ has problems with statically linking exception code.
+  # If linking fails, remove -static.
   LINKFLAGS = -static -std=c++11
```

Comment:
For the static compilation problems, see [Multiple definition error in static compilation with g++ under cygwin](https://stackoverflow.com/questions/46854365/multiple-definition-error-in-static-compilation-with-g-under-cygwin)